swatch-system-conduit/src/main/java/org/candlepin/subscriptions/conduit/inventory/InventoryService.java

```diff
@@ -168,6 +170,16 @@ public OrgInventory getInventoryForOrgConsumers(List<ConduitFacts> conduitFactsF
     return rhsmFactMap;
   }
 
+  private SystemProfileOperatingSystem buildOS(String name) {
+    var operatingSystems = new SystemProfileOperatingSystem();
```

Comment:
```suggestion
   if(!operatingSystemName.toLowerCase().contains("red hat enterprise linux")){
      return null;
   }
   var operatingSystems = new SystemProfileOperatingSystem();
   operatingSystems.setName(SystemProfileOperatingSystem.NameEnum.RHEL);
   return operatingSystems;
  }
```

Also, if there's any potential for operatingSystemName to be null, we'd want to null check that before `.toLowerCase()`